agent/exec/dockerapi/controller_test.go

```diff
@@ -14,7 +14,6 @@ import (
 
 	"github.com/docker/docker/api/types"
 	"github.com/docker/docker/api/types/container"
-	containertypes "github.com/docker/docker/api/types/container"
```

Comment:
nice catch on these 👍 

still wondering what linter we don't have enabled that would catch these (I'm sure there's a linter somewhere in golangci-lint that should warn about these)